README.md

```diff
@@ -100,14 +100,14 @@ HUGO_UGLYURLS=false hugo serve -w
 > **Note:**  
 > "Ugly URLs" refer to URLs ending with `.html` (e.g., `/about.html`). GitHub Pages prefers these for compatibility. The `serve-public.js` script emulates this behavior.
 
-Pro-Tip: Do this in a sparse checkout that excludes large parts of `content/`, to speed up the rendering time.
```

Comment:
This pro-tip is now removed? The end result might be clearer, and at the same time worse because helpful hints have been removed without adequate replacement.